SB2018/2018-02-27.md

```diff
@@ -0,0 +1,26 @@
+---
+to: All Teams
+subject: SourceBots Competiton dates confirmed! (And some other bits)
+---
+
+Hello all!
+
+I'm happy to confirm the dates for the competition this year! The competition will be held from *Saturday 21st April* until *Sunday 22nd April*. The competition will be held at the University of Southampton's Highfield campus, so please plan accommodation / transport accordingly. The exact start/end times are to be confirmed, but it'll start at some point in the morning of the 21st, and finish in the afternoon of the 22nd, no later than 5:00pm.
+
+This information will be reflected on our website soon!
+
+## Competition Progress
+
+You're now over half way through this competition period. Hopefully you're making good progress with your robot, and are well on your way to having something competition ready!
+
+## Tech Days
+
+There are just 2 remaining tech days between now and the competition. These are a great chance to spend a full day testing on your kit, and get direct support from us. The remaining tech days are:
+
+- 17th March
+- 14th April
+
+Both of these are likely to be held at Highfield campus, as with the last 2, however confirmation of this will come closer to the time.
```

Comment:
Is it implied that Highfield Campus is at Southampton University? Not sure if we should assume this knowledge for teams who both didn't come to kickstart or any previous tech days.

Reply:
2 -> two (unrelated to above comment)